src/bun.js/webcore/response.zig

```diff
@@ -3708,6 +3709,11 @@ pub const Body = struct {
                     result.status_code = @truncate(u16, @intCast(u32, number));
             }
 
+            if (response_init.fastGet(ctx, .status_text)) |text| {
+                var str = text.toSlice(ctx, allocator);
+                result.status_text = str.slice();
```

Comment:
- JS string may be freed at any point in the future so here we will need to duplicate it. This leaks memory
- doesn't validate HTTP status text (ascii text, no newlines or carriage return. probably safest to limit to letters, numbers, underscores, dashes and spaces)